Tests/RoslynPluginGeneratorTests/AnalyzerPluginGeneratorTests.cs

```diff
@@ -307,6 +308,98 @@ public void Generate_InvalidSqaleFileSpecified_GeneratorError()
             logger.AssertSingleErrorExists("invalidSqale.xml"); // expecting an error containing the invalid sqale file name
         }
 
+        [TestMethod]
+        public void CreatePluginManifest_TitleMissing()
```

Comment:
Only the properties that have had additional mapping behaviour beyond "if available then set" are tested.